src/postgres/commands/executePostgresQuery.ts

```diff
@@ -56,4 +56,5 @@ export async function executePostgresQuery(context: IActionContext): Promise<voi
 
         await vscodeUtil.showNewFile(csvData, outputFileName, '.csv');
     }
+    await treeItem.refresh();
```

Comment:
Some words of wisdom given to myself to be aware of: https://github.com/microsoft/vscode-azurestorage/pull/840#discussion_r499943538.

I believe using a refresh in this case is OK though because it doesn't interfere with the file system (we open Postgres queries as one-off new files)